packages/api/lib/rulesHelpers.js

```diff
@@ -468,11 +469,10 @@ function updateKinesisRuleArns(ruleItem, ruleArns) {
    */
 async function addRule(item, payload) {
   const name = `${process.env.stackName}-custom-${item.name}`;
-  const state = item.enabled ? 'ENABLED' : 'DISABLED';
   await CloudwatchEvents.putEvent(
     name,
     item.rule.value,
-    state,
+    item.state,
```

Comment:
Added unit test to cover this function https://github.com/nasa/cumulus/pull/2728/files#diff-f5c5b96c6230da09644104acbdca2268e22d32509bc5b275a5b4648a8939584eR409

Reply:
👍🏽